node/src/neighborhood/neighborhood_database.rs

```diff
@@ -378,10 +385,28 @@ mod tests {
 
     #[test]
     fn a_brand_new_database_has_the_expected_contents() {
-        let this_node = make_node_record(1234, true);
+        let mut this_node = make_node_record(1234, true);
+        this_node.inner.country_code = "AU".to_string();
+        this_node.metadata.node_location_opt = Some(NodeLocation {
+            country_code: "AU".to_string(),
+            free_world_bit: true,
+        });
+        this_node.resign();
 
         let subject = db_from_node(&this_node);
 
+        let this_pubkey = this_node.public_key();
+        let last_update = subject
+            .by_public_key
+            .iter()
+            .map(|(pubkey, node_record)| match pubkey == this_pubkey {
+                true => node_record.metadata.last_update,
+                false => todo!("implement me"),
+            })
+            .exactly_one()
+            .unwrap();
```

Comment:
Here you can easily use this notation:

```Rust
let last_update = subject.root().metadata.last_update;
```
It's because you know there is only one record at the moment, the root Node. 

This is much better solution, also solving the problem with the left over todo!().